app.js

```diff
@@ -22,4 +23,23 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res, send)=>{
+  
+  punkAPI.getBeers()
+  .then((getBeers) => {
+    console.log(getBeers);
+    res.render('beers', {getBeers:getBeers})
+  })
+  .catch((error => {
+    console.log(error)
+  }));
+})
+
+app.get('/random-beer', (req, res, send) => {
+  punkAPI.getRandom()
+  .then((beerFromApi)=>{
+    res.render('random-beer', {beerFromApi})
```

Comment:
Here I would pass the element directly, that way you could skip the `#each` block in the random-beer.hbs

```suggestion
    res.render('random-beer', beerFromApi[0])
```